G4HepEm/G4HepEm/include/G4HepEmTrackingManager.hh

```diff
@@ -22,13 +23,13 @@ class G4HepEmWoodcockHelper;
 class G4HepEmTrackingManager : public G4VTrackingManager {
 public:
   G4HepEmTrackingManager();
-  ~G4HepEmTrackingManager();
+  virtual ~G4HepEmTrackingManager();
 
   void BuildPhysicsTable(const G4ParticleDefinition &) override;
 
   void PreparePhysicsTable(const G4ParticleDefinition &) override;
 
-  void HandOverOneTrack(G4Track *aTrack) override;
+  virtual void HandOverOneTrack(G4Track *aTrack) override;
```

Comment:
This function is already `virtual` from `G4VTrackingManager` and is marked `override` here. The C++ core guidelines recommend only having one of `virtual`, `override`, (and `final`) in any class member declaration.

Reply:
Of course, thanks for spotting! Fixed now